### PR TITLE
add pre change document for submission deletion

### PR DESCRIPTION
**Pull request by sblack-usu**
In order for the discovery trigger that is listening to submission changes to delete a discovery record after a submission is deleted the changeStreamPreAndPostImages option must be enabled on the Submission collection. https://www.mongodb.com/docs/manual/reference/command/collMod/#change-streams-with-document-pre--and-post-images

Before this update when submissions were deleted the discovery collection was not updated because the trigger could not access the record id.  This is an edge case that was somehow missed in our tests.

### CODE REVIEW

File: management/change_streams_pre_and_post.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+async def main():
+    db = AsyncIOMotorClient(get_settings().mongo_url)
+    await init_beanie(database=db[get_settings().mongo_database], document_models=[Submission])
+    # https://www.mongodb.com/docs/manual/reference/command/collMod/#change-streams-with-document-pre--and-post-images
+    # This enables us to get the record id of a deleted submission within a trigger.  We need this for removing
+    # discovery records when a submission is deleted
+    await db[get_settings().mongo_database].command(
+        ({'collMod': Submission.get_collection_name(), "changeStreamPreAndPostImages": {'enabled': True}}))
+
```
Comment (pkdash) [inline management/change_streams_pre_and_post.py:17]: Should the db connection be closed? `db.close()`

**Review by pkdash — APPROVED** [review]

File: management/change_streams_pre_and_post.py (hunk 1 above)
Comment (sblack-usu) [reply management/change_streams_pre_and_post.py:18]: yes it should, updated.
